File: JAVA/7thOct-5.java
import java.lang.*; 
import java.io.*; 
import java.util.*;

public class Main
{
public static void main(String[] args)
	{
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter an integer number: ");
        String str= sc.nextLine(); 
        int len=str.length();
        char c[]=str.toCharArray();
        String reverse="";
        //For loop to reverse a string
        for(int i=c.length-1;i>=0;i--)
        reverse+=c[i];

		if (str.charAt(0)=='-')
		{
			System.out.println("Enter numbers greater than 0");
		}
		else
		{
		    
		    String[] numeral=new String[len];
		    for (int i =len-1; i>=0; i--) 
		    {
              char letter = reverse.charAt(i);
              
              if(i==0)
                {
                  numeral[0]= "" +letter;}
              if(i==1)
               {
                   if(letter=='1')
                   numeral[1]= letter +" ten and ";
                   else
                   numeral[1]= letter +" tens and ";
                   
               }
              if(i==2)
                {
                   if(letter=='1')
                   numeral[2]= letter + " hundred ";
                   else
                   numeral[2]= letter + " hundreds ";
                }
              if(i==3)
                {
                   if(letter=='1')
                   numeral[3]= letter + " thousand ";
                   else
                   numeral[3]= letter + " thousands ";
                }
              if(i==4)
              {
              numeral[4]= "" +letter;}
              if(i==5)
              {
                   if(letter=='1')
                   numeral[5]= letter + " Lac ";
                   else
                   numeral[5]= letter + " Lacs ";
              }
              if(i==6)
              {
              numeral[6]= "" +letter;}
              if(i==7)
              {
                   if(letter=='1')
                   numeral[7]= letter + " Crore ";
                   else
                   numeral[7]= letter + " Crores ";
              }
              else if(i>7)
              numeral[i]= "" + letter;
              
            }
            for(int j=len-1;j>=0;j--)
              System.out.print(numeral[j]);

		}
	}
}

File: Python/7thOct-5.py
n=input()
l=list(n)
i=0
while(i<len(l)):
    if(i==len(l)-1):
        print("and",end=' ')
    if len(l[i+1:])==8:
        print("{} crore".format(l[i]+l[i+1]),end='')
        i=i+2
    elif len(l[i+1:])==7:
        print("{} crores".format(l[i]),end=' ')
        i=i+1
    elif len(l[i+1:])==6:
        print("{} lacs".format(l[i]+l[i+1]),end=' ')
        i=i+2
    elif len(l[i+1:])==5:
        print("{} lac".format(l[i]),end=' ')
        i=i+1
    elif len(l[i+1:])==4:
        print("{} thousands".format(l[i]+l[i+1]),end=' ')
        i=i+2
    elif len(l[i+1:])==3:
        print("{} thousands".format(l[i]),end=' ')
        i=i+1
    elif len(l[i+1:])==2:
        print("{} hundreds ".format(l[i]),end=' ')
        i=i+1
    elif len(l[i+1:])==1:
        print("{} tens".format(l[i]),end=' ')
        i=i+1
    elif len(l[i+1:])==0:
        print("{}".format(l[i]),end=' ')
        i=i+1
        
